03week/news/index.html

```diff
@@ -6,8 +6,8 @@
     <!--This is my style sheet-->
     <link rel="stylesheet" type="text/css" href="css/style.css">
```

Comment:
I noticed your page doesn't have a title. Probably not a big deal for this assignment but wanted to call it out.